[PR-22316] Add a Tremendous Integration page (#338)
talkable/talkable-docs: source/advanced_features/tremendous.rst

.. _advanced_features/tremendous:
.. include:: /partials/common.rst

.. meta::
   :description: Reward advocates with gift cards provided by Tremendous.

Tremendous Integration
######################

Reward advocates with gift cards from various retailers provided by Tremendous.

.. note::

   The Tremendous integration is an enterprise feature.

>    To become an enterprise client, please `contact Talkable team <mailto:support@talkable.com>`_.

Reviewer: Lets have it consistent with other "Contact us" blocks:

```
**Contact us**

Interested in setting this up? Contact your CSM or get in touch `here <https://lp.talkable.com/lets-talk-referral>`_.
```

Tremendous integration consists of several parts:

  1. A campaign in Tremendous with the selected rewards for customers.

  3. A "Tremendous" application installed from Talkable App Store. This application is responsible

>      for storing API tokens so that Talkable can request gift cards from Tremendous and insert the generated

Reviewer: I'd change it to 
```suggestion
     for storing API tokens, enabling Talkable to request gift cards from Tremendous and insert the generated
```
